charts/stolon/values.yaml

```diff
@@ -121,6 +123,16 @@ proxy:
 observability:
   enabled: false
 
+exporter:
+  enabled: false
+  image:
+    repository: quay.io/prometheuscommunity/postgres-exporter
+    tag: v0.15.0
+    pullPolicy: IfNotPresent
+  datasourceSecret: {}
```

Comment:
I think it is useless option. We'll use just `DATA_SOURCE_URI` env variable

Reply:
resolved